frontend/packages/console-dynamic-plugin-sdk/scripts/create-pkg.ts

```diff
@@ -0,0 +1,26 @@
+import * as fs from 'fs';
+import * as readPkg from 'read-pkg';
+
+const PKG_NAME = '@openshift-console/dynamic-plugin-sdk';
+const EXTENSIONS_EXPORT = 'lib/extensions/index.js';
+const WEBPACK_PLUGIN_EXPORTS = './lib/webpack/ConsoleRemotePlugin.js';
+
+const createPackageJson = () => {
+  const packageJson = readPkg.sync({ normalize: false });
+  packageJson.name = PKG_NAME;
+  delete packageJson.private;
+  packageJson.license = 'Apache-2.0';
+  packageJson.main = EXTENSIONS_EXPORT;
+  packageJson.exports = {
+    './': `./${EXTENSIONS_EXPORT}`,
+    './webpack': WEBPACK_PLUGIN_EXPORTS,
```

Comment:
We should create a `public` folder convention for all our packages such that everything in the `public` dir is exposed api so it is very clear when we change API.
```
"exports": {
  "./*": "./src/public/*"
}
```